src/modules/tables/dto/create-table.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsBoolean, IsDefined, IsString } from 'class-validator';
+
+export class CreateTableDto {
+  @ApiProperty({ description: 'The table name', required: true })
+  @IsString({ message: 'table_name_required' })
+  name: string;
+  @ApiProperty({
+    description: 'The table active-inactive status',
+    required: true,
+  })
+  @IsDefined({ message: 'table_is_active_attribute_required' })
```

Comment:
Why do you define the attribute isActive? When you create a table is always active, dont you?

Reply:
Ok, I think that it is reasonable i'm going to change it